db/migrate/20160226014344_create_pallets.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePallets < ActiveRecord::Migration
+  def change
+    create_table :pallets do |t|
+      t.references :cycle_count, index: true, foreign_key: true
+      t.string :pallet_number, null: false
+      t.boolean :pallet_in_grossman, default: false
+      t.boolean :pallet_in_location, default: false
```

Comment:
I think I may be a little confused, was it your intent to use combinations of the two flags to determine the state of the pallet?

```
# would is_in_location be used when not in grossman?
is_in_grossman=0
is_in_location=?
```

```
# found in grossman, but the locations do not match
is_in_grossman=1
is_in_location=0
```

```
# found in grossman, and the locations match
is_in_grossman=1
is_in_location=1
```

Please confirm, personally I don't like this but its probably not worth beating any further.

My main concern and question would be what happens when the person wants to know where the pallet is in Grossman?